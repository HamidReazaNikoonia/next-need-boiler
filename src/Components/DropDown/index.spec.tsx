// #region Global Imports
import * as React from "react";
import { mount } from "enzyme";
// #endregion Global Imports

// #region Local Imports
import { DropDown } from "@Components";
// #endregion Local Imports

describe("DropDown", () => {
    it("should match snapshot", () => {
        const wrapper = mount(<DropDown />);
        expect(wrapper).toMatchSnapshot();
    });
});
