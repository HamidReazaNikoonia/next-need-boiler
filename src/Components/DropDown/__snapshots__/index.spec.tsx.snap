// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDown should match snapshot 1`] = `ReactWrapper {}`;
